### PR TITLE
MOTECH-2695: Add element id naming conventions to email module

### PR DESCRIPTION
**Pull request by jhopen**

### CODE REVIEW

File: src/email/send.html [hunk 1]
```diff
@@ -1,35 +1,35 @@
...
     <legend class="frame-title">{{'email.compose.title' | translate}}</legend>
     <fieldset>
         <div class="form-group">
-            <label for="fromAddress">{{'email.from.address' | translate}}</label>
-            <input type="email" ng-model="mail.fromAddress" id="fromAddress" name="fromAddress" aria-describedby="emailForm-fromAddress-errors" required />
-            <ul id="emailForm-fromAddress-errors" class="form-hint">
-                <li ng-if="emailForm.$submitted && emailForm.fromAddress.$error.required ">{{'email.field.required' | translate }} {{'email.from.address' | translate}}</li>
-                <li ng-if="emailForm.$submitted && emailForm.fromAddress.$error.email ">{{'email.field.notValid' | translate}} {{'email.from.address' | translate}}</li>
+            <label for="email.send.from-address">{{'email.from.address' | translate}}</label>
+            <input id="email.send.from-address" type="email" ng-model="mail.fromAddress" name="fromAddress" aria-describedby="emailForm-fromAddress-errors" required />
```
Comment (nickdotreid) [inline src/email/send.html]: @jhopen -- the ID on the `<ul>` element is for accessibility, and is referenced by the aria-describedby property // please make sure this matches...

Can we change the format for the error message IDs? something like "email.send.from-address.errors.required"

The `<li>` element ids are missing an "s"

Comment (jhopen) [inline src/email/send.html]: @nickdotreid -- fixed
- sure we can do that, it will require do update elements ID naming convention in doc.
- fixed

Comment (nickdotreid) [inline src/email/send.html]: @jhopen -- I'll make a point of getting the docs updated

**Comment by sebbrudzinski** [comment]
@nickdotreid Did you have a chance to take a look at the PR after fixes?

**Comment by nickdotreid** [comment]
@sebbrudzinski thanks for bringing this back up... it looks good and I merged it.... I will follow up with Piotr in the related ticket to see if this fits his needs....

**Pull request merged**